llama_parse/utils.py

```diff
@@ -100,60 +100,93 @@ class Language(str, Enum):
 
 
 SUPPORTED_FILE_TYPES = [
-    ".pdf",
-    # Microsoft word - all versions
+    # document and presentations
+    ".602",
+    ".abw",
+    ".cgm",
+    ".cwk",
     ".doc",
     ".docx",
     ".docm",
     ".dot",
-    ".dotx",
```

Comment:
we removed a few file types (by accident? pdf was removed for example) -- will try to add back

Reply:
I thiiiiink it was just .pdf missing lol